leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.kt

```diff
@@ -111,12 +115,21 @@ class GeocoderPresenter @JvmOverloads constructor(
         val disposable = geocoderRepository.getFromLocation(latLng)
                 .observeOn(scheduler)
                 .retry(RETRY_COUNT.toLong())
-                .subscribe({ view!!.showLocationInfo(it) },
+                .filter { addresses -> !addresses.isEmpty() }
+                .map { addresses -> addresses[0] }
+                .flatMap { address -> returnTimeZone(address) }
+                .subscribe({ pair: Pair<Address, TimeZone?> -> view!!.showLocationInfo(pair) },
```

Comment:
It's not going to raise null pointer exceptions because it's using the nullview pattern. Maybe I need to remake this part to avoid using these unwrappings. Not in the scope of this PR but a good catch. Thanks!